makefile

```diff
@@ -2,7 +2,7 @@
 .PHONY: all release clean ontology mappings mappings-validate help
 TODAY ?=$(shell date +%Y-%m-%d)
 VERSION=v$(TODAY)
-SOURCE_URL=https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
+SOURCE_URL=https://icd11files.blob.core.windows.net/foundationowl/whofic-2024-01-21.owl.gz
```

Comment:
## Hold off until we determine syncing branches
- [ ] 1. Sabrina and ICD11 folks to conclude which branches
- [ ] 2. Then @joeflack4  can merge this if it is otherwise approved

Context, [Nico wrote](https://monarchinitiative.slack.com/archives/C03AW548J8M/p1716706052588129?thread_ts=1716600006.410389&cid=C03AW548J8M):
> before the pipeline is re-run with the new ICD11 Foundation url, i'd like to get agreement with Sabrina and answers from the ICD11 folks about sections we should not try to map Mondo to.